### ISSUE TITLE
4.5 second sleep upon initialization

### ISSUE THREAD

**Issue by egorikem** [AppsFlyerSDK/flutter_appsflyer_sdk#22]
**Describe the bug**
There is a 4.5-second sleep in the library initialization:
```dart
 Future<Map> initSdk(bool registerConversionDataCallback,bool registerOnAppOpenAttributionCallback) async {
    sleep(const Duration(milliseconds: 4500));
    ....
}
```

**To Reproduce**
1. Use SDK

**Expected behavior**
There is no delay in SDK initialization

**Info**
```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, v1.15.19, on Microsoft Windows [Version 10.0.18362.719], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Chrome - develop for the web
[√] Android Studio (version 3.6)
[√] VS Code (version 1.39.1)
[√] Connected device (2 available)

• No issues found!
```
`appsflyer_sdk: 1.2.2`

**Comment by egorikem** [comment]
Come on, guys, this is so bad!

**Comment by vagrantrobbie** [comment]
what is the sleep for? If we know what it's for, we can fix it ourselves and submit a pull request

**Comment by rostopira** [comment]
What the actual fuck?

**Comment by rostopira** [comment]
If delay is actually needed, you should use `Future.delayed` instead of blocking the thread

**Comment by egorikem** [comment]
Are you serious, guys? It's been a **WEEK** since this totally critical issue and still no response? What's the use of your multiple emails to me asking "did we fix it" if you don't actually fix any issues with your library?

**Comment by ShaharAF** [comment]
I've released now a new version without the sleep and now will work on a workaround to fix the initialisation issue.
thanks for the feedback

**Comment by ShaharAF** [comment]
Changed this to `Future.delayed`. working fine on iOS, still there are some issues on Android. 
released new version `1.2.5`
